docs/dev_contribution.rst

```diff
@@ -0,0 +1,95 @@
+.. Contribution documentation for libCellML
+
+============
+Contributing
+============
+
+This document covers the process to follow for getting your changes into the :term:`prime repository`.  While there are many types of contribution, this section focuses on contributions made through GitHub and Git, or in other words assets that are managed using the version control system.  It is assumed that :doc:`Setup <dev_setup>` and :doc:`Building <dev_building>` have already been read and followed. 
+
+.. contents::
+
+Overview
+========
+
+For any body of work intended for the :term:`prime repository` start with a GitHub issue.  The issue can be used to discuss the topic and clarify any problems related to it.  Once progress has been made towards addressing the issue a pull request is created that references the issue.  
+
+Reviewers provide feedback on the changes by adding comments to the pull request or associated commits. The BuildBot build/test procedure will run each time changes are pushed to the pull request's branch, and the results are displayed in the pull request view.
+
+Once all the changes and reviews are complete, one of the :term:`prime repository` owners will merge the pull request into the prime repository, onto the develop branch.
+
+Note that a bug is just a type of issue, and that resolving the bug should have both the implementation to fix the bug and a test that triggers the bug.
+
+:numref:`Figure %s <libcellml-development-process>` gives a graphical overview of the developer contribution process.  For more detail see the text below.
+
+.. _libcellml-development-process:
+.. figure:: images/libCellMLProcesses-DevelopmentProcess.png
```

Comment:
A couple of very minor points:
 * You might want to replace **cellml/libcellml** with **CellML/libCellML**.
 * Missing full point at the end of **Reviewer to merge pull request**.